sympy/functions/special/gamma_functions.py

```diff
@@ -160,6 +160,13 @@ def _eval_is_real(self):
         if x.is_positive or x.is_noninteger:
             return True
 
+    def _eval_is_positive(self):
+        x = self.args[0]
+        if x.is_positive:
+            return True
+        elif x.is_noninteger:
```

Comment:
should we add `elif x.is_nonpositive and x.is_integer: return False`? Otherwise this looks ok but could use a test like 
```
assert gamma(Symbol('e', even=True, negative=True) + S.Half).is_positive is True
assert gamma(Symbol('e', even=True, negative=True) - S.Half).is_positive is False
```